Ekwipunek.py

```diff
@@ -215,16 +221,19 @@ class Pozostale(Przedmiot):
     def __init__(self, nazwa: str, wartosc: int, ilosc: int):
         super().__init__(nazwa, wartosc, ilosc)
 
-
+# Generalna sprawa. Ekwipunek powinien mieć kontenery: Bronie, Magiczne etc. do nich na podstawie typu dodajesz odpowiedni przedmio.
+# Używaj isinstance! Możesz użyć danej klasy kontenera jako klucz słownika w_użyciu ( generalny koncept, pomyśl jak to dobrze zrobić )
```

Comment:
Czyli jeśli dobrze rozumiem, to kontenerów (słowników) ma być tak dużo, jak danych klas, tak? I później dodaję wszystko do jednego, wielkiego słownika jakim jest magazyn